src/lib/components/dashboard/credentials/CredentialList.svelte

```diff
@@ -37,6 +38,7 @@
   let noCardsSelected = false;
   let showCreateCredentialModal = false;
   let isShareHovered = false;
+  let accesslistHovered = false;
 
   $: sortedCredentials = $credentialStore.sort(
```

Comment:
can you move this to the storeHelper. there we are setting the credential store. we dont need to use sorted credentials.